text/0630-new-browser-support-policy.md

```diff
@@ -0,0 +1,67 @@
+- Start Date: 2020-05-21
+- Relevant Team(s): All
+- RFC PR: https://github.com/emberjs/rfcs/pull/630
+- Tracking: (leave this empty)
+
+# New Browser Support Policy
+
+## Summary
+
+> In dropping support for IE11 at some point in the future, we will need a new
+browser support policy. This is the proposed policy.
+
+## Motivation
+
+> It should be clear which browsers are supported by Ember.js
+
+## Detailed policy
```

Comment:
For this entire section, I would simply say "Browser support" and then list all of the browsers. If you want to further indicate mobile devices, put that in parenthesis after the list item.

Reply:
Given that mobile browsers are being given significantly different treatment from desktop browsers, this is impractical.

Reply:
Can you explain more? I disagree that it's impractical, but maybe you have more information I'm missing.

Reply:
Browser support for Firefox on Desktop but not for mobile. Browser support for Android browser is mobile only, no desktop support.

For all other browsers, browser support is also much longer for mobile than desktop because browsers on mobile are typically only updated as long as phone vendors choose to support the phones which means that we have to support Chrome, Safari, and Android for a full 3 years.

If we list browsers first and then make exceptions for mobile, we make exceptions for every single browser.